.travis.yml

```diff
@@ -150,9 +150,19 @@ branches:
   only: [master]
 
 # MANUALLY ADDED. Do not remove.
-after_success:
-  - ./tool/deploy.sh
-# END MANUAL EDITS.
+deploy:
```

Comment:
Actually does the work to install the tools – and only when we want them installed!